packages/devextreme/js/__internal/ui/tree_view/tree_view.base.ts

```diff
@@ -95,6 +95,8 @@ export interface TreeViewBaseProperties extends Properties<TreeViewNode>, Omit<
   deferRendering?: boolean;
 
   _supportItemUrl?: boolean;
+
+  showItemStubs?: boolean;
```

Comment:
It’s possible to implement a solution without using a stub element at all, but since this is a bug and would require refactoring the current implementation, we’ll skip that option. 

I’d prefer the variant that simply hides the stub element, as is done in other components that customize TreeView appearance. That way, we only need to add one selector with display none for the TreeView in the menu.